lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/robot.cpp

```diff
@@ -4,15 +4,13 @@
 // https://docs.google.com/spreadsheets/d/1eX79YtawJqBA8VePFFtKJT6RR21gvK4qH1DKJX_vJE0/edit#gid=0
 
 // MCs
-Sabertooth MC1(128, ST_SERIAL); // DRIVE
-Sabertooth MC2(130, ST_SERIAL); // EXCAVATON
-Sabertooth MC3(131, ST_SERIAL); // DEPOSIT
+Sabertooth MC1(128, ST_SERIAL); // LINEAR ACTUATORS
 
 // ADCs
 namespace actuation {
 
-Sabertooth_MotorCtrl act_right_mtr{&MC3, STMotor::M2};
-Sabertooth_MotorCtrl act_left_mtr{&MC2, STMotor::M2};
+Sabertooth_MotorCtrl act_right_mtr{&MC1, STMotor::M1};
+Sabertooth_MotorCtrl act_left_mtr{&MC1, STMotor::M2};
 
 constexpr uint8_t ACT_RIGHT_CURR_ADC = 0;
```

Comment:
These will almost certainly need to be updated. We probably won't have mutliple ADCs for only two sensors